### ISSUE TITLE
Linear slide code

### ISSUE THREAD

**Issue by 12sliu** [Brighton-FTC/2024#8]
The arm will have a linear slide on it, so the grabber can extend to score; this will be done with a small motor.

**Comment by 12sliu** [comment]
We've confirmed that it exists, please write code it for it, thanks!

**Comment by NatalieMountain** [comment]
Hi! I have completed a first draft for the linear slide code - I could not find the functionality in FTC-Lib for a joystick so I have coded for buttons, if anyone else has had luck with that could they please let me know? Thankyou so much!

**Comment by 12sliu** [comment]
Your changes have not been committed and pushed; we cannot see them as is. 

A guide:
1. Open Github Desktop.
2. Make sure you are in the correct repository (2024) and the correct branch (the one starting with fredi).
3. Look through your changes; there should be a new file added with lots of green code (indicating it's newly added).
4. Write a title for your commit.
5. Press commit button.
6. There is a blue button saying "push" or something similar; press that button too.

Refer to the wiki on Github in the future, as it should contain advice on how to commit and push.

**Comment by 12sliu** [comment]
@NatalieMountain, please appear at a emergency session as soon as you can so I can go through your code with you and explain what you did wrong. You aren't in trouble, but I need to speak to you asap.

**Comment by NatalieMountain** [comment]
Hi! Unfortunately, I am unable to attend todays p10 session but I belive I can attend Wednesday's - does that work for you?

**Comment by 12sliu** [comment]
Yes; if I don't appear at whatever location you are by 4:00, send me a email to remind me, because I'll be at the FRC place.

**Comment by sbottingota** [comment]
https://github.com/Brighton-FTC/2024/commit/c9ccd70ca42dd3aeae038526aa8c6361c09a7539

Converted Fredi's code into a component class.

https://github.com/Brighton-FTC/2024/commit/00e00a1b68f031a5c693c80000773cf7b40bf4db

Added position finder and functionality tester.

**Comment by 12sliu** [comment]
Still needs to be better tuned after the engineers install the arm, and we need to add a F value.
